doc/source/whatsnew/v1.5.1.rst

```diff
@@ -83,7 +83,7 @@ Fixed regressions
 - Fixed :meth:`.DataFrameGroupBy.size` not returning a Series when ``axis=1`` (:issue:`48738`)
 - Fixed Regression in :meth:`DataFrameGroupBy.apply` when user defined function is called on an empty dataframe (:issue:`47985`)
 - Fixed regression in :meth:`DataFrame.apply` when passing non-zero ``axis`` via keyword argument (:issue:`48656`)
--
+- Fixed regression in Stata reading where files were needlessly buffered in memory (:issue:`48922`)
```

Comment:
Not sure whether it should go in 1.5.1 or in the next major release.

Reply:
Mm, how so? Even if the bug/regression is pretty old, I think this is still a simple bugfix.

Reply:
Since the bug was reported way back in https://github.com/pandas-dev/pandas/commit/2f0ada3d430f0cc49fa9ab1e7e2e2a7ec23c9616, I don't think this can be considered a recent regression so I would suggest moving to 1.6.0.rst

Reply:
@mroeschke If you really think so. I just imagine 1.6.0's release might be quite far off, and [there evidently are people who could use this fix](https://stackoverflow.com/a/73934594/51685)...

Reply:
pandas has purposed point releases to fix regressions and bugs occurring from the last minor release.

While all fixes are useful in theory in a next release, it has been good to keep the point releases minimal like this to minimize introducing a new bug.